functions/_fzf_search_pids.fish

```diff
@@ -0,0 +1,19 @@
+function _fzf_search_pids --description "Search pid for all running commands"
+    set fzf_arguments --multi --ansi $fzf_pids_opts
+    set token (commandline --current-token)
+    # expand any variables or leading tilde (~) in the token
+    #set expanded_token (eval echo -- $token)
+
+    set pid_selected (
+        ps -A -opid,command | \
+        _fzf_wrapper --query $token  \
+                     --preview='ps -f -p {1}' | \
```

Comment:
Sorry I was unclear. I didn't mean an explicit fzf preview, though that's nice. I mean it'd be good to show everything that the user needs in fzf directly so that it can be skimmed and filtered on by fzf's search.

Reply:
I actually think it's less useful that way...
Or rather, the current implementation simply puts the pid and command line (and relegates the other details for the preview), I think just adding columns such as parent pid are less useful.
What are your thoughts?

Reply:
You're right--those things aren't that useful for searching and would clutter the view. I'm thinking the user will narrow down the process to a few matching the name and then use the preview to find the exact process to operate on. In that case, since the preview window offers tons of space and it's not expensive to put more info there, can you find a few more fields that might be of interest to the user?

Reply:
looking at how I've been using activity monitor, maybe memory use, CPU use, IO use?

Reply:
Hmm I see that idea is complicated by this: https://github.com/dalance/procs#permissions-issues. Do you see a need to be able to kill processes not owned by the current user? If not, I wonder if we should not show processes owned by other users.

Reply:
Not sure we can find good portable flags for that. 
Generally, I've taken the view of using the `-f` flag which offers a sane default. 
Not sure what we'd want to add there.

As for processes by the user only, I thought about it, but at least in my use-case I quite often do something like `sudo kill XXX`. 

One option would be to check the full command and see if it begins with sudo, or if the euid is root and change behavior based on that. WDYT?

Reply:
Or maybe use commandline --current-process to extract the last token and see if it begins with sudo. 
I fear it's a bit fragile...

Reply:
> Generally, I've taken the view of using the -f flag which offers a sane default.

Ok that seems fair.

> One option would be to check the full command and see if it begins with sudo, or if the euid is root and change behavior based on that. WDYT?

I dont' want to get into the business of parsing the user's command. And if we're not going to try displaying memory and disk use, then there's no point in doing it.

Reply:
Cool. Dropping